spec/truefi2/BorrowingMutex.spec

```diff
@@ -23,9 +30,8 @@ rule functionDoesNotBanUnlockedBorrower() {
     require isReasonableEnv(e);
```

Comment:
This stays as is, as returning structs in functions is not yet supported by Certora.